js1/funkce-arg-udalosti/funkce-arg.md

```diff
@@ -75,21 +75,19 @@ Ve Velké Británii by však očekávali datum spíš jako 25/04/2023:
 
 ```js
 const formatGb = (date) => {
-  return `${String(date.day).padStart(2, '0')}/${String(date.month).padStart(
-    2,
-    '0'
-  )}/${date.year}`;
+  const day = String(date.day).padStart(2, '0');
```

Comment:
Chápu motivaci této úpravy, taky jsem nad tím přemýšlel, jestli to tak už neudělat. Přidržel jsem se tehdy toho původního stavu, kdy to bylo jednořádkové. Každopádně v reálném projektu bych já osobně tyhle proměnné nazýval třeba formattedDay nebo paddedDay nebo podobně, protože nemám rád, když v blízkém kontextu existuje nějaké "day" (na tom "date"), které je číslo, a jiné "day" tady ta proměnná, která nejenom, že je jiného datového typu, řetězec, ale ještě s nějakou úvodní nulou. Nevím, jestli to takhle komplikovat i v akademii, ale v praxi bych to tak dělal. (Samozřejmě totéž s tím měsícem a u té druhé formátovací funkce.)

Reply:
Mě to až tak zásadní nepřijde, pokud ta proměnná má takhle krátkou platnost. Ale opravil jsem to pro lepší čitelnost